samples/csharp/events/Program.cs

```diff
@@ -62,13 +62,13 @@ internal SearchDirectoryArgs(string dir, int totalDirs, int completedDirs)
     }
     public class FileSearcher
     {
-        public event EventHandler<FileFoundArgs> OnFoundFile;
-        internal event EventHandler<SearchDirectoryArgs> OnChangeDirectory
+        public event EventHandler<FileFoundArgs> FoundFile;
```

Comment:
I think the noun should come first and then the verb. So it would be FileFound event and DirectoryChanged event.
Some examples: [ControlRemoved](http://msdn.microsoft.com/en-US/library/ms613658(VS.110).aspx), [SystemColorsChanged](http://msdn.microsoft.com/en-US/library/ms614408(VS.110).aspx), [UserPreferenceChanging](http://msdn.microsoft.com/en-US/library/yy9zxw67(VS.110).aspx), etc.

@terrajobst might know all the rules since he drives the framework design discussions.